ratatui-core/Cargo.toml

```
@@ -0,0 +1,70 @@
+[package]
+name = "ratatui-core"
+description = """
+    Core library for the Ratatui terminal UI framework. This crate contains the core types and
+    traits that are used to build terminal user interfaces. Most applications should use the
+    `ratatui`, while widget libraries should use this crate."""
```

We should have a shorter description here and move the explanation to the README.md in `ratatui-core`. What do you think?

---

This shows up in search results, Ithink that should include the note about widget libs. Eventually this lib will be the second highest ratatui lib to show up by downloads due to being always included when downloading Ratatui.

This definitely does need a README too though.

---

How about making it a bit shorter?

```suggestion
description = """
Core library for the Ratatui terminal UI framework, containing essential types and traits for building terminal UIs. Most applications should use the `ratatui`, while widget libraries should use this crate."""
```

---

Simplfied in 8709cdb7d..a8ff86830

Stylistically, I avoid using backticks around crate names unless actually pointing at the crate's module (This is mostly consistent in Ratatui, but there's a few places where that isn't the case).